resources/code/lf-emr-workshop-js.template

```diff
@@ -547,7 +547,7 @@
           "Fn::Join": [
             "-",
             [
-              "notebooks",
+              "lf-notebooks-bucket",
```

Comment:
Please use lf-scripts-bucket as discussed

Reply:
Sure will re-factor this bucket creation after the workshop next week.